src/main/java/controller/Controller.java

```diff
@@ -0,0 +1,51 @@
+package controller;
+
+import Interface.InputBoundary_h_u;
+import javafx.beans.InvalidationListener;
+
+import java.util.Scanner;
+
+public class Controller implements InputBoundary_h_u {
+    /**
+     * The method will return the player decide to upgrade sword or armor.
+     * @param EquipmentChoice
+     * @return return the choice keyword [A][S]
```

Comment:
Perhaps we should change the key for sword to just [W] for weapons so that it can be generalized to different types of weapons